tools/js-reference/lib/lib.js

```diff
@@ -0,0 +1,34 @@
+const crypto = require('crypto');
```

Comment:
> npm WARN deprecated crypto@1.0.1: This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.

```suggestion
const crypto = require('node:crypto');
```

built-in since v16 (minimum we require); tests passing